src/main/java/seedu/address/model/person/ModuleTakenPredicate.java

```diff
@@ -0,0 +1,40 @@
+package seedu.address.model.person;
+
+import java.util.List;
+import java.util.Set;
+import java.util.function.Predicate;
+
+import seedu.address.model.tag.NusModule;
+
+/**
+ * Tests if any of a {@code Person}'s {@code Module} matches any of the keywords given.
+ */
+public class ModuleTakenPredicate implements Predicate<Person> {
+    private final List<NusModule> modules;
+
+    public ModuleTakenPredicate(List<NusModule> modules) {
+        this.modules = modules;
+    }
+
+    @Override
+    public boolean test(Person person) {
```

Comment:
This will match all Persons who take *at least one* of the modules listed. Is this the intended behavior?

Reply:
It is, according to the JavaDoc. Closing.